frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/LitigantHomeConfig.js

```diff
@@ -298,14 +279,15 @@ export const TabLitigantSearchConfig = {
           uiConfig: {
             columns: [
               {
-                label: "Draft Name",
+                label: "Case Name",
```

Comment:
here as well

Reply:
Not done here since this value is being mapped in ui customisation. Hence localised value is affecting the expected output